### DISCUSSION TITLE
Enqueue song multiple times

### DISCUSSION THREAD

**Discussion by BRTPOB** [audacious-media-player/audacious#1487] [Q&A]
I'm just getting started with Audacious after years of using XMMS/XMMS2 way back when, and having used foobar2000 via wine in the interim. The question I have is, is there a way of enqueuing a song multiple times in a row? Right now it looks like it's either on or off, you can't have a song queued 2+ times in a row.

Is that something that's an advanced feature I'm missing, or is that a feature request?

**Comment by radioactiveman** [comment]
As far as I'm aware, this is not supported. You can only enqueue a track once.

You can of course copy & paste the track in the playlist. But that is obviously not really convenient.

**Comment by BRTPOB** [comment]
So should I turn this into a feature request, since it's not currently supported?

**Reply by radioactiveman** [reply]
Sure, feel free to do that.

**Comment by Fr-Dae** [comment]
> I'm just getting started with Audacious after years of using XMMS/XMMS2 way back when, and having used foobar2000 via wine in the interim. The question I have is, is there a way of enqueuing a song multiple times in a row? Right now it looks like it's either on or off, you can't have a song queued 2+ times in a row.
> 
> Is that something that's an advanced feature I'm missing, or is that a feature request?

what are you distrib /OS, because i don't understand why you use audacious with *wine*

**Comment by BRTPOB** [comment]
I don't use Audacious with wine. I use(d) foobar2000 with wine.

**Reply by Fr-Dae** [reply]
> I don't use Audacious with wine. I use(d) foobar2000 with wine.

I don't understand why this was inflicted on you.
Audacious and MPV are native to Linux and often installed by default (especially on lubuntu). 

espacially audacious have a contextual menu (right clic on file or folder) => play with audacious

**Reply by mschwendt** [reply]
> I don't understand why this is happening.

Visit https://www.foobar2000.org/

**Reply by BRTPOB** [reply]
> > I don't use Audacious with wine. I use(d) foobar2000 with wine.
> 
> I don't understand why this is happening. Audacious and MPV are native to Linux and often installed by default (especially on lubuntu).
> 
> espacially audacious have a contextual menu (right clic on file or folder) => play with audacious

I don't understand what you're confused by. I was asking if functionality, which exists in foobar2000, existed in Audacious. I clearly (to me) stated that I used foobar2000 via wine. I never once said I was using Audacious through wine.

I know what Audacious and MPV are, and I'm fully aware that they're native which is one of the reasons I mentioned using XMMS/XMMS2 in yesteryear.

**Reply by Fr-Dae** [reply]
translation issue, edited
